src/utils/emojifyFunc.ts

```diff
@@ -0,0 +1,151 @@
+const rI = 'regional_indicator_'; // Prefix for regional indicators, just to save on space.
+
+const fallBackEmoji = ':joker:'; // Wildcard, baby :)
+
+const zWJ = '\u200D'; // This is required to stop emoji from making little emoji babies. Otherwise :regional_indicator_c::regional_indicator_a: would become 🇨🇦
```

Comment:
I figure this should paste into Discord fine, but I also have it as a character here >> ‌ <<